cmd/zetaclientd-supervisor/lib.go

```diff
@@ -169,6 +177,124 @@ func (s *zetaclientdSupervisor) watchForVersionChanges(ctx context.Context) {
 	}
 }
 
+func (s *zetaclientdSupervisor) handleTSSUpdate(ctx context.Context) {
```

Comment:
consider using `bg.Work` wrapper as it handles possible panics

Reply:
That's a good idea; however, I am planning to remove the functions from the supervisor and add them to the zeta client in the next PR; I'll keep this suggestion in mind when implementing that.